Building Applications with React and Flux.md

```diff
@@ -0,0 +1,15 @@
+# This is a development environment for React
+
+#### With React comming to Free Code camp soon. I thought this would be a aset to the community.
+
+ > This is designed to run in the cloud nine environment.
```

Comment:
~~designed to run in the cloud nine environment.~~ designed to run in a c9 environment.